metron-platform/metron-writer/src/test/java/org/apache/metron/writer/BulkWriterComponentTest.java

```diff
@@ -118,6 +118,20 @@ public void writeShouldProperlyAckTuplesInBatch() throws Exception {
   }
 
   @Test
+  public void writeShouldProperlyAckTuplesWhenWriterDisabled() throws Exception {
```

Comment:
The first one (writer for one sensor is enabled and writer for another sensor is disabled).  Should have been more clear.